dss-xades/src/main/java/eu/europa/esig/dss/xades/XAdESIdPrefixes.java

```diff
@@ -0,0 +1,141 @@
+/**
+ * DSS - Digital Signature Services
+ * Copyright (C) 2015 European Commission, provided under the CEF programme
+ *
+ * This file is part of the "DSS - Digital Signature Services" project.
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+package eu.europa.esig.dss.xades;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class XAdESIdPrefixes {
+	private static final String ID_PREFIX = "id-";
+	private static final String TIMESTAMP_PREFIX = "ts-";
+	private static final String ENCAPSULATED_TIMESTAMP_PREFIX = "ets-";
+	private static final String KEYINFO_PREFIX = "keyInfo-";
+	private static final String VALUE_PREFIX = "value-";
+	private static final String XADES_PREFIX = "xades-";
+
+	/** Xml Id prefix */
+	private final String id;
```

Comment:
We might want to apply some restrictions on the allowed characters for a prefix so that generated ids can't break the XML document - or should this be responsibility of the application using the API?

Reply:
I added some simple illegal xml character validation